x-pack/plugins/translations/translations/fr-FR.json

```diff
@@ -25449,7 +25449,7 @@
     "xpack.securitySolution.eventsViewer.unit": "{totalCount, plural, =1 {événement} other {événements}}",
     "xpack.securitySolution.exceptions.dissasociateListSuccessText": "La liste d'exceptions ({id}) a été retirée avec succès",
     "xpack.securitySolution.exceptions.exceptionItem.showCommentsLabel": "Afficher {comments, plural, =1 {commentaire} other {commentaires}} ({comments})",
-    "xpack.securitySolution.exceptions.failedLoadPolicies": "Une erreur s'est produite lors du chargement des politiques : \"{error}\"",
+    "xpack.securitySolution.exceptions.failedLoadPolicies": "Une erreur s'est produite lors du chargement des politiques : \"{error}\"",
```

Comment:
Do you know why this has changed?

Reply:
Good catch. That looks odd! let me run the script again and update this

Reply:
Okay I reran the scripts and it's the same set of changes. I think these changes are probably due to line shifts.